README.md

```diff
@@ -1,5 +1,157 @@
+# What is the application?
+REST API monolith Spring application that uses a postgres database to store customers. A REST API is exposed to the user. 
+
+### Endpoints
+
+1. **Get all customers**
+    - **Request:**
+        - Method: GET
+        - URL: `/api/v1/customers`
+          - Example Request:
+            ```
+            GET /api/v1/customers
+            ```
+
+    - **Response:**
+        - Status Code: 200 OK
+        - Example Response:
+          ```
+          [
+          {
+            "id" : 1
+            "name": "Alice"
+            "email": "Alice@gmail.com"
+            "age": 30
+          },
+           {
+           "id": 2,
+           "name": "Bob",
+           "email": "bob@example.com",
+           "age": 30
+           },
+           {
+            "id": 3,
+            "name": "Charlie",
+            "email": "charlie@example.com",
+            "age": 28
+            }
+          ]
+          ```
+
+2. **Add a new customer**
+    - **Request:**
+        - Method: POST
+        - URL: `/api/v1/customers`
+        - Example Request:
+          ```
+          POST /api/v1/customers
+          Content-Type: application/json
+   
+          {
+            "name": "John Doe",
+            "email": "john@example.com",
+            "age": 30
+          }
+          ```
+
+    - **Response:**
+        - Status Code: 201 Created
+        - Example Response:
+          ```
+          {
+            "id" 1
+            "name": "John Doe",
+            "email": "john@example.com",
+            "age": 30
+          }
+          ```
+
+3. **Delete a customer**
+    - **Request:**
+        - Method: DELETE
+        - URL: `/api/v1/customers/{customerId}`
+        - Example Request:
+          ```
+          DELETE /api/v1/customers/123
+          ```
+
+    - **Response:**
+        - Status Code: 200 OK
+      - Example Response:
+         ```
+           {"message": "Customer with ID {id} was deleted successfully."}
```

Comment:
Make sure the curly braces are on separate lines. ID is `123` as that's what you put as your example request.